packages/capabilities/src/access.js

```diff
@@ -74,33 +74,62 @@ export const authorize = capability({
 })
 
 /**
- * Issued by trusted authority (usually the one handling invocation that contains this proof) 
- * to the account (aud) to update invocation local state of the document.
+ * Capability is delegated by us to the user allowing them to complete the
+ * authorization flow. It allows us to ensure that user clicks the link and
+ * we don't have some rogue agent trying to impersonate user clicking the link
+ * in order to get access to their account.
+ */
+export const confirm = capability({
+  can: 'access/confirm',
+  with: DID,
+  nb: Schema.struct({
+    iss: Account,
+    aud: Schema.did(),
+    att: CapabilityRequest.array(),
+  }),
+  derives: (claim, proof) => {
+    return (
+      fail(equalWith(claim, proof)) ||
+      fail(equal(claim.nb.iss, proof.nb.iss, 'iss')) ||
+      fail(equal(claim.nb.aud, proof.nb.aud, 'aud')) ||
+      fail(subsetCapabilities(claim.nb.att, proof.nb.att)) ||
+      true
+    )
+  },
+})
+
+/**
+ * Issued by trusted authority (usually the one handling invocation) that attest
+ * that specific UCAN delegation has been considered authentic.
  *
- * @see https://github.com/web3-storage/specs/blob/main/w3-account.md#update
+ * @see https://github.com/web3-storage/specs/blob/main/w3-session.md#authorization-session
  * 
  * @example
  * ```js
  * {
     iss: "did:web:web3.storage",
-    aud: "did:mailto:alice@web.mail",
+    aud: "did:key:z6Mkk89bC3JrVqKie71YEcc5M1SMVxuCgNx6zLZ8SYJsxALi",
     att: [{
-      with: "did:web:web3.storage",
-      can: "./update",
-      nb: { key: "did:key:zAgent" }
+      "with": "did:web:web3.storage",
+      "can": "ucan/attest",
+      "nb": {
+        "proof": {
+          "/": "bafyreifer23oxeyamllbmrfkkyvcqpujevuediffrpvrxmgn736f4fffui"
+        }
+      }
     }],
     exp: null
     sig: "..."
   }
  * ```
  */
 export const session = capability({
-  can: './update',
+  can: 'ucan/attest',
```

Comment:
would be ideal to link to relevant spec for this, e.g. so someone could figure out how to build a valid/useful `Link` for `.nb.proof`

Reply:
The link on line 105 add this, or do you mean something else ?